docs/source/api/turbulence.rst

```diff
@@ -0,0 +1,11 @@
+***********************************
+Turbulence Time Series Calculations
+***********************************
+
+
+:mod:`metpy.calc.turbulence`
+========================
```

Comment:
Pretty sure if you build this, Sphinx will complain that the `====` don't extend long enough. Needs to match line above.